.eslintrc.js

```diff
@@ -11,13 +10,19 @@ module.exports = {
     browser: true,
     node: true,
   },
-  extends: ['plugin:import/recommended', 'airbnb', 'prettier', 'prettier/react'],
-  parser: 'babel-eslint',
+  extends: [
+    'plugin:import/recommended',
+    'plugin:import/typescript',
+    'airbnb-typescript',
+    'prettier',
+    'prettier/react',
+    'prettier/@typescript-eslint',
+  ],
+  parser: '@typescript-eslint/parser',
   parserOptions: {
-    ecmaVersion: 7,
```

Comment:
Why was this removed?

Reply:
I don't know why it was here in the first place. In the unknown and with a green CI, I think that it's better to relying on the default configuration.

Reply:
Green CI is not necessarily a good indicator for changes in the lint configuration. As far as I can tell this would now allow later ecma features (after 7). Even if this was the default I'd rather be explicit about it.